blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+```
+
+
+### Jupyter notebook
+After you have Memgraph running, you should install **Jupyter**. For this tutorial, we used **jupyter-lab** and you can install it with the following command:
+```
+pip install jupyterlab
+```
+
+Now you can go and download **[jupyter-notebook](https://github.com/memgraph/jupyter-memgraph-tutorials/tree/main/cugraph-analytics)** we use in our tutorial. Note, in a repository folder `cugraph-analytics/data` there are `CSV` files containing the full dataset which we will use later in the tutorial.
+
+
+### GQLAlchemy installation
+We will be using the **[GQLAlchemy](https://memgraph.com/docs/gqlalchemy/)** object graph mapper (OGM) to connect to Memgraph and execute **Cypher** queries from **Python**. You can think of Cypher as SQL for graph databases. It contains many of the same language constructs like `CREATE`, `UPDATE`, `DELETE`... and it's used to query the database. GQLAlchemy serves as a Python driver/client for Memgraph. Go to our [installation](https://memgraph.com/docs/gqlalchemy/installation) page to check how more options on how to install it. If you wish, you can install it with `pip`, but be sure also to have `CMake` installed. After you have `CMake` ready, you can install `GQLAlchemy` with the following command:
+```
+pip install gqlalchemy
+```
+
+### Memgraph Lab installation
+Last piece of tech you will use is **[Memgraph Lab]()**. You will use it to connect to **Memgraph** and create visualizations. To check how to install **Memgraph Lab** and connect to **Memgraph**, you can go to one of the following pages for installation instructions for **Memgraph Lab as desktop application** for [Windows](https://memgraph.com/docs/memgraph-lab/installation/windows#step-2---installing-and-setting-up-memgraph-lab), [Mac](https://memgraph.com/docs/memgraph-lab/installation/macos#step-2---installing-and-setting-up-memgraph-lab) or [Linux](https://memgraph.com/docs/memgraph-lab/installation/linux#step-2---installing-and-setting-up-memgraph-lab). After you have installed **Memgraph Lab**, you can check instructions how to **[connect to Memgraph](https://memgraph.com/docs/memgraph-lab/connect-to-memgraph#connecting-to-memgraph)** with Memgraph Lab. 
+
+
+Now, when you have installed everything, we continue to the main topic of the tutorial: **importing large dataset** and **running graph analytics** with **Python**.
+
+## Connecting to Memgraph with GQLAlchemy
+
+Now position yourself in [jupyter-notebook](https://github.com/memgraph/jupyter-memgraph-tutorials/tree/main/cugraph-analytics) you previously downloaded. We will start with the `qglalchemy` import and then we will connect to **Memgraph** using `host` and `port`. Afterward, we will clear our database, just in case.
+
+
+```python
+from gqlalchemy import Memgraph
+```
+
+
+```python
+memgraph = Memgraph("127.0.0.1", 7687)
+```
+
+
+```python
+memgraph.drop_database()
+```
+
+In the next few steps we will do the following:
```

Comment:
Lets import the dataset from CSV files and learn how to  perform `PageRank` and `Louvain community detection` using **Python**